src/SitemapExtension.php

```diff
@@ -219,4 +222,15 @@ private function linkIsIgnored($link)
         // No absolute match & no regex match
         return false;
     }
+
+    /**
+     * @param MutableBag $links
+     * @return MutableBag
+     */
+    private function transformByListeners($links)
+    {
+        $event = new SitemapEvent($links);
+        $this->getContainer()['dispatcher']->dispatch(SitemapEvents::AFTER_COLLECTING_LINKS, $event);
+        return $event->getLinks();
```

Comment:
It would be good to have empty line before return

Reply:
Fix'd :)